src/app/customer-search/customer-search.component.ts

```diff
@@ -40,7 +39,8 @@ export class CustomerSearchComponent implements OnInit {
     // audiologist-navigation to change state
     public loadAppt(appt: Appointment) {
         if (appt['id'] === '') { return; }
-        // for each dataService saveData
+        Utilities.setSessionStorage('dataFromDB', 'true');
```

Comment:
I don't understand what's going on here at cursory glance here but it looks pretty janky, could you explain?

Reply:
This is a merging of Elijah's and my changes.
The if statement is now pointless, since it was changed to take an Appointment.
Looking everything over, I got rid of the whole dataFromDB session key.  It wasn't obvious what it really meant anyway.  So now this function just emits an appointment that the audiologist-navigation component sees.